src/App.tsx

```diff
@@ -61,13 +62,19 @@ export const App = (): React.ReactElement => {
                   </NavLink>
                 </Nav.Item>
                 <Nav.Item>
-                  <NavLink
-                    to="/paper"
-                    className="nav-link"
-                    activeClassName="active"
-                  >
-                    White Paper
-                  </NavLink>
+                  <NavDropdown title="White Paper" id="basic-nav-dropdown">
+                    <NavDropdown.Item href="/paper">
+                      Table of Contents
+                    </NavDropdown.Item>
+                    {Object.keys(pages).map((pageId, pageIndex) => (
+                      <NavDropdown.Item
+                        href={`/paper/${pageId}`}
+                        key={pageIndex}
+                      >
+                        {pages[pageId].shortTitle}
```

Comment:
Can we add the numbers here to more obviously distinguish the TOC from the subpages? Like `1: Introduction` or similar.

Reply:
Good idea! On it!

Reply:
Done.